plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -3482,6 +3482,75 @@ public void testUpdatingInvalidTableProperty()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testEmptyCreateTableAsSelect()
+    {
+        String tableName = "test_empty_ctas_" + randomTableSuffix();
+
+        assertUpdate("CREATE TABLE " + tableName + " WITH (format = '" + format.name() + "') AS SELECT * FROM nation WHERE nationkey < 0", "VALUES 0");
```

Comment:
nit: I would use just `0` instead of `"VALUES 0"` as the 2nd argument. Other places are also same.